src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -19,7 +21,10 @@ const AppContent = React.memo(({ repo, repoVersions, onRequestUpgrade }) => {
     name,
     instances,
     baseUrl,
-    currentVersion,
+    currentVersion: {
+      content: { details_url: detailsUrl },
```

Comment:
We should use the details from the `latestVersion` instead, so you can see the new description when upgrading.